targets/xfce-desktop

```diff
@@ -7,8 +7,25 @@ DESCRIPTION='Installs Xfce along with common applications. (Approx. 1200MB)'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Avoid /etc/skel/.xscreensaver conflicts
-xsvr='/etc/skel/.xscreensaver'
-[ -f "$xsvr" ] && mv -f "$xsvr" "$xsvr.bak"
-install ubuntu=xubuntu-desktop,task-xfce-desktop -- network-manager xorg
-[ -f "$xsvr.bak" ] && mv -f "$xsvr.bak" "$xsvr"
+
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    # Use this target to install the rest of the xfce4 packages and more
+    install orage mousepad ristretto squeeze thunar-archive-plugin \
```

Comment:
Gentoo is kind of minimal, it expects you to install the stuff you need. There are meta-packages, but they're not as encompassing as something like the *-desktop targets here.

That said, if you'd rather not have a giant list of packages here, I could make an overlay (kind of like a ppa) with Gentoo meta-package ebuilds that do the same thing.